### DISCUSSION TITLE
Bluetooth Classic

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#24143] [Q&A]
Hello,
 I need to send command through bluetooth classic (as serial) to an HC-05 Module, and receive the response. from MAUI android app. Basically it's an arduino thermometer, I send "T" and should receive the temperature. My problem is that I can't find any plugin/library to allow that, it seem that all are for BLE only; Isn't there any way to do that?

**Comment by acaliaro** [comment]
You can try https://github.com/inthehand/32feet

**Reply by ghost** [reply]
I already tried it but without success, but I could make another try.
